bindings/python/tools/python/ostk/astrodynamics/viewer.py

```
@@ -16,14 +16,18 @@
 
 from ostk.mathematics.geometry.d3.transformation.rotation import Quaternion
 
+from ostk.physics import Environment
```

Ruff (0.8.2)

`ostk.physics.Environment` imported but unused

Remove unused import: `ostk.physics.Environment`

(F401)